src/deluge/storage/storage_manager.h

```diff
@@ -112,6 +110,8 @@ class StorageManager {
 	SyncLevel readAbsoluteSyncLevelFromFile(Song* song);
 	void writeAbsoluteSyncLevelToFile(Song* song, char const* name, SyncLevel internalValue, bool onNewLine = true);
 
+private:
+public:
```

Comment:
are these supposed to be private?

Reply:
Renaming the storage manager class to shake-out unexpected global refs is definitely part of the plan for the next phase. I did get rid of the private/public issue.